### ISSUE TITLE
[Form Validation] Adding/Removing/Clearing/Replacing Validations

### ISSUE THREAD

**Issue by douglasg14b** [Semantic-Org/Semantic-UI#3568]
Feature is necessary for forms where the javascript state is maintained between form submits, and when using dynamic forms where certain fields are added/removed and where their validation can change.

Heck, even easy access to whatever object the validation is stored in would be great so we can remove or change the validations directly.

**Comment by pareeohnos** [comment]
I'm interested in a nice way of doing this as well. For now I've got the following which might help you:

    $(form).form(...rules...);
    $(form).form('destroy');
    $(form).form(...new rules...);

According to #3058 it should be possible to simply override the `fields` setting with the new fields, however I have been unable to get this to work. I found that this simply added the new fields, but did NOT add the validations for those fields. Hopfully @jlukic can shed some light on this?
